package.json

```diff
@@ -14,9 +14,9 @@
     "build:docs:monetary-amount": "jsdoc2md lib/monetary-amount.js > Docs/monetary-amount.md",
     "build:docs:utils": "jsdoc2md lib/utils.js > Docs/utils.md",
     "build:docs:erc20": "jsdoc2md lib/erc20/erc20-contract.js > Docs/erc20-contract.md",
-    "build:docs:settlement": "jsdoc2md lib/settlement/settlement.js > Docs/settlement.md",
-    "build:docs:driip-settlement": "jsdoc2md lib/settlement/driip-settlement.js > Docs/driip-settlement.md",
-    "build:docs:null-settlement": "jsdoc2md lib/settlement/null-settlement.js > Docs/null-settlement.md",
+    "build:docs:payment-settlement": "jsdoc2md lib/settlement/payment-settlement.js > Docs/payment-settlement.md",
+    "build:docs:onchain-balance-settlement": "jsdoc2md lib/settlement/onchain-balance-settlement.js > Docs/onchain-balance-settlement.md",
+    "build:docs:settlement-factory": "jsdoc2md lib/settlement/settlement-factory.js > Docs/settlement-factory.md",
```

Comment:
These document files should also be linked into the README.md like the other files.